appengine-managed-runtime/src/test/java/com/google/apphosting/vmruntime/VmApiProxyDelegateTest.java

```diff
@@ -321,13 +352,21 @@ private void callDelegateWithParsingError(boolean sync, ApiProxyException expect
     byte[] requestData = new byte[] {0, 1, 2, 3, 4, 5};
     byte[] result = null;
 
+    Logger logger = Logger.getLogger(VmApiProxyDelegate.class.getName());
```

Comment:
Also, did you consider putting it in a private method that would set the Logger and return the previous level?